packages/core/integration-tests/test/elm.js

```diff
@@ -83,6 +84,10 @@ describe('elm', function () {
   });
 
   it('should produce correct formatting and indentation when compilation fails', async function () {
+    const normalizedPath = normalizePath(
+      'test/integration/elm-compile-error/src/Main.elm',
+      false,
+    );
```

Comment:
I'm sorry, `normalizePath` is actually the other direction (to always use `/`).
This should be `path.normalize('test/integration/elm-compile-error/src/Main.elm')`

Reply:
Ah, I just came to the same conclusion after looking at the source code of  `normalizePath`  :-D no worries.
I made the change and pushed. Hopefully this fixes the test.
Thanks for you pointers.